src/api/java/com/minecolonies/api/util/InventoryUtils.java

```diff
@@ -1,7 +1,6 @@
-package com.minecolonies.coremod.util;
+package com.minecolonies.api.util;
```

Comment:
Overruled. Utility Classes are supposed to be treated this way, and as such are God Classes by Definition.